app/assets/stylesheets/admin/base.scss

```diff
@@ -1,6 +1,42 @@
+$eff-red: #ec1e1e;
+$gray: #949494;
 a:link,
 a:visited,
 a:focus,
 a:active {
   color: #00d;
-}
\ No newline at end of file
+}
+
+/* Tooltip container */
+.tooltip {
+  color: $gray;
+  border: 2px solid $gray;
+  border-radius: 1rem;
+  display: inline-block;
+  font-size: 18px;
```

Comment:
I've been sticking to em/rem units for font-size (16px = 1rem).

Reply:
Oh, me too, thanks for catching that.